WORKSPACE.bazel

```diff
@@ -55,3 +57,42 @@ scalatest_toolchain()
 ## https://github.com/bazelbuild/rules_scala/issues/1521
 ## Workaround is to use our own scala toolchain with custom options.
 register_toolchains("//toolchains:enso_toolchain")
+
+
+## Setup `bazel-distribution` for generating pom.xml
+RULES_BAZEL_DISTRIBUTION_SHA = "3f88eb2ff11654039fa64cbc67a8411df1b85fba"
+
+git_repository(
+    name = "vaticle_bazel_distribution",
+    commit = RULES_BAZEL_DISTRIBUTION_SHA,
+    remote = "https://github.com/vaticle/bazel-distribution",
+)
+
+### Dependencies of `bazel-distribution`
+http_archive(
+    name = "rules_pkg",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.9.1/rules_pkg-0.9.1.tar.gz",
+        "https://github.com/bazelbuild/rules_pkg/releases/download/0.9.1/rules_pkg-0.9.1.tar.gz",
+    ],
+    sha256 = "8f9ee2dc10c1ae514ee599a8b42ed99fa262b757058f65ad3c384289ff70c4b8",
+)
+## Uses more up-to-date version of rules_kotlin than what
+## @vaticle_bazel_distribution seems to rely on
+## as encountered various issues, including a blocker of
+## https://github.com/bazelbuild/rules_kotlin/issues/1072.
+rules_kotlin_version = "1.9.0"
```

Comment:
Why do we need kotlin rules?

Reply:
Dependency of `bazel-distribution` unfortunately.